cpp/test/cpu_operations_test/multiply_test.cc

```diff
@@ -1,53 +1,42 @@
-#include <iostream>
+// Copyright 2016 <Morgan Rockett>
```

Comment:
All set

On Thu, Jun 23, 2016 at 9:25 PM, yiskylee <notifications@github.com> wrote:

> In cpp/test/cpu_operations_test/multiply_test.cc
> <https://github.com/yiskylee/NICE/pull/23#discussion_r68342212>:
>
> > @@ -1,53 +1,42 @@
> > -#include <iostream>
> > +// Copyright 2016 <Morgan Rockett>
>
> This should be the MIT License, you can copy and paste the license from
> src/core/cpu_operations.cc
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/yiskylee/NICE/pull/23/files/9e8eb25e29087e171732fb3e7756ddf5bcd410cb..d0b6fd6195dde3213146f81ae4752c833ac36cff#r68342212>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARN57phBHFd_NpC_SVdrOGaYr4-84j8Zks5qOzHsgaJpZM4I8NOH>
> .
>